go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/nginxinc/nginx-go-crossplane
 
 go 1.19
+toolchain go1.24.1
```

Comment:
this broke the linter action, might need to bump that first

https://github.com/nginxinc/nginx-go-crossplane/blob/main/.github/workflows/cicd.yml#L31

Reply:
https://github.com/nginxinc/nginx-go-crossplane/pull/137